include/mega/megaclient.h

```diff
@@ -511,6 +511,11 @@ class MEGA_API MegaClient
     // keep sync configuration after logout
     bool mKeepSyncsAfterLogout = false;
 
+    // backup names pending to be sent
+    string_map mPendingBackupNames;
+
+    // true if setting the backup name for any backup id is in progress
+    bool mSendingBackupName = false;
```

Comment:
why not check emptyness of mPendingBackupNames?

Reply:
It cannot be done, since we may have pending backup names while we are sending others, and need to decide whether to send the new backup names directly, or not, depending on the current state: sending or not, rather than empty map (which may or may not accumulate new names).